app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -159,8 +159,9 @@ class Scene extends React.Component {
     // directly on top of each other.
     World.add(
       this.engine.world,
-      Bodies.circle(this.state.width / 2 + (Math.random() - 0.5), -50, 17, {
-        restitution: 0.7,
+      Bodies.circle(this.state.width / 2 + (Math.random() - 0.5), -50, 8, {
```

Comment:
might be a good idea to squash c56b48b15934e0ec8d1ecf0cc501e36a6e63190e into this commit, as this changes the size of the circle but doesn't touch the texture.